shared/constants/types/flux.js

```diff
@@ -3,11 +3,13 @@
 export type TypedAction<T, P, E> = {
   error?: false,
   type: T,
-  payload: P
+  payload: P,
+  skipLogging?: boolean,
```

Comment:
if this is slow just while you're devving in chrome maybe this isn't something we really care about anymore?